docs/chart_best_practices/conventions.md

```diff
@@ -4,14 +4,14 @@ This part of the Best Practices Guide explains general conventions.
 
 ## Chart Names
 
-Chart names should be lower case letters and numbers. Words _may_ be separated with dashes (-):
+Chart names should be lower case letters and numbers. Dashes (-) are not allowed:
 
 Examples:
 
 ```
-drupal
-nginx-lego
-aws-cluster-autoscaler
+drupal #correct
+cluster01 #correct
+aws-cluster-autoscaler #incorrect do not use dashes in the name
```

Comment:
can we remove the `#correct` comments and this line altogether, only showing what examples are good to use?